server/conf/i18n/messages

```diff
@@ -564,3 +564,10 @@ banner.govWebsiteSectionHeader=Official websites use .gov
 banner.govWebsiteSectionContent=Website addresses ending in .gov belong to official government organizations in the United States.
 banner.httpsSectionHeader=Secure .gov websites use HTTPS
 banner.httpsSectionContent=A lock ( {0} ) or https:// means you''ve safely connected to the .gov website. Only share sensitive information on official, secure websites.
+
+#----------------------------------------------------------------------------------------#
+# NOT PRODUCTION BANNER - text explaining that this site is not a production environment #
+#----------------------------------------------------------------------------------------#
+
```

Comment:
**suggestion:** Could you add context strings in comments above each of the strings here? Just for translators to give them more context.

Reply:
Are you trying to get me to train Gemini? :P 

Done